src/@dvcorg/gatsby-theme-iterative/components/MainLayout/base.css

```diff
@@ -96,3 +104,7 @@ body.user-is-tabbing .link-with-focus {
 * {
   text-size-adjust: none;
 }
+
+[class^='styles-module--innerSidebar'] {
+  @apply pt-20;
+}
```

Comment:
This one could be the kinda hacky way but I didn't want to shadow the entire CSS file for a single change.